### DISCUSSION TITLE
DS3231 and i2c

### DISCUSSION THREAD

**Discussion by rwl7532** [jkotlinski/durexforth#533] [Q&A]
I found a utility to read the date and time from the DS3231 RTC via I2C by way of the user port.
[Instagram pics](https://www.instagram.com/p/CqLoH4TBZTC/?igshid=YmMyMTA2M2Y=) 
Has anyone done durexforth i2c and read the DS3231?

**Comment by Whammo** [comment]
The most I have done with the userport and durexForth is blink LEDs.

**Reply by rwl7532** [reply]
https://github.com/gnacu/i2c6502
Done in assembly. I'm not knowledgeable in it.
